oneview/resource_logical_interconnect_group.go

```diff
@@ -1218,9 +1218,7 @@ func resourceLogicalInterconnectGroupUpdate(d *schema.ResourceData, meta interfa
 	for i := 0; i < uplinkSetCount; i++ {
 		uplinkSetPrefix := fmt.Sprintf("uplink_set.%d", i)
 		uplinkSet := ov.UplinkSets{}
-		if val, ok := d.GetOk(uplinkSetPrefix + ".name"); ok {
```

Comment:
is this code not required?

Reply:
It is required. Have removed it by mistake while I was comparing the resource for another branch. Added it back now. Thanks for pointing it out.